app/views/admin/calendar/_legend.html.slim

```diff
@@ -4,20 +4,23 @@ div#legend-content
       div.column.col-lg-6
         table.legend-block
           tr
+            td
```

Comment:
Draft steps do not exist in the main calendar. Do we need to have a revised list of legend items for the general calendar or is that not needed?

Reply:
I don't think it's a big deal.

Reply:
Ok.